src/index.html

```diff
@@ -10,23 +10,35 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Search Bar</title>
     <link
       rel="stylesheet"
       href="style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      data-qa="big"
+      class="form"
+    >
+      <input
+        data-qa="keypress"
+        type="text"
+        placeholder="Try “Los Angeles“"
+        class="form__input form__input--primary"
+      />
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      data-qa="small"
+      class="form"
```

Comment:
It's a good practice to use the `action` attribute in a `form` tag. Even if you are not sending the form data anywhere, you can set `action="#"` to indicate that the form doesn't have a specific action page.